### ISSUE TITLE
no forward attachment eml in filters

### ISSUE THREAD

**Issue by ilya1986r** [roundcube/roundcubemail#9133]
Hi, 
How enable to forward attachment eml in filters for Roundcube Webmail 1.6.2?
By default, there is no such option forward attachment eml in filters.
![Uploading filters.jpg…]()

**Comment by alecpl** [comment]
Could you be more precise? I don't understand. There's definitely a forwarding feature in the managesieve plugin ("Redirect message to", "Send message copy to" actions).

**Comment by ilya1986r** [comment]
yes, "Redirect message to", "Send message copy to" actions are present, but these actions do not allow redirecting the message in eml. Do you know how to enable forwarding in eml in actions?

**Comment by ilya1986r** [comment]
> Could you be more precise? I don't understand. There's definitely a forwarding feature in the managesieve plugin ("Redirect message to", "Send message copy to" actions).

yes, "Redirect message to", "Send message copy to" actions are present, but these actions do not allow redirecting the message in eml attachment. Do you know how to enable forwarding in eml attachment in actions?

**Comment by alecpl** [comment]
There's no such option in the standard. And I do not recall any extension that would add this.

https://www.rfc-editor.org/rfc/rfc3028#section-4.3
